epimodel/exports/epidemics_org.py

```diff
@@ -170,25 +175,31 @@ def extract_smallish_data(
                     self.region.Code,
                     nulls.to_dict(),
                 )
-            d["JohnsHopkins"] = {
+            data["JohnsHopkins"] = {
                 "Date": [x.date().isoformat() for x in hopkins.index],
                 **hopkins.astype("Int64").replace({pd.NA: None}).to_dict(orient="list"),
             }
 
         if foretold is not None:
-            d["Foretold"] = {
+            data["Foretold"] = {
                 "Date": [x.isoformat() for x in foretold.index],
                 **foretold.replace({np.nan: None})
                 .loc[:, ["Mean", "Variance", "0.05", "0.50", "0.95"]]
                 .to_dict(orient="list"),
             }
 
-        d["Timezones"] = timezones["Timezone"].tolist()
+        data["Timezones"] = timezones["Timezone"].tolist()
 
         if un_age_dist is not None:
-            d["AgeDist"] = un_age_dist.to_dict()
+            data["AgeDist"] = un_age_dist.to_dict()
 
-        return d
+        if r_estimates is not None:
+            data["REstimates"] = {
```

Comment:
FYI: For the frontend we have [typescript typing's describing the data exchange format](https://github.com/epidemics/covid/blob/master/common/spec/v4.ts).

It would be very useful to add this field here: https://github.com/epidemics/covid/blob/master/common/spec/v4.ts#L24-L30